src/vm/win32threadpool.cpp

```diff
@@ -348,9 +348,23 @@ BOOL ThreadpoolMgr::Initialize()
     //ThreadPool_CPUGroup
     CPUGroupInfo::EnsureInitialized();
     if (CPUGroupInfo::CanEnableGCCPUGroups() && CPUGroupInfo::CanEnableThreadUseAllCpuGroups())
+    {
         NumberOfProcessors = CPUGroupInfo::GetNumActiveProcessors();
+    }
     else
-        NumberOfProcessors = GetCurrentProcessCpuCount();
+    {
+        int cpuCount = GetCurrentProcessCpuCount();
+        int cpuLimit = GetCurrentProcessCpuLimit();
+        _ASSERTE(1 <= cpuLimit && cpuLimit < INT_MAX);
+
+        if (cpuLimit < cpuCount)
```

Comment:
I have thought that based on the @VSadov's comment on the threadpool  (https://github.com/dotnet/coreclr/issues/22302#issuecomment-475817427), we would not limit the cpu count for threadpool.

Reply:
Fixed that with https://github.com/dotnet/coreclr/pull/23398/files#diff-35149ba730fd61aba699e375e72c5f3fR2528